### PR TITLE
support for ILOG parameters in solveCOBRALPCPLEX as in #728

### PR DESCRIPTION
**Pull request by marouenbg**
*Please include a short description of enhancement here*
Added support for ILOG cplex parameters in solveCOBRALPCPLEX , using the function of PR #728
**I hereby confirm that I have:**

- [] Tested my code on my own machine
- [] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [] with [X] to check the box)*

### CODE REVIEW

File: src/base/solvers/solveCobraLPCPLEX.m [hunk 1]
```diff
@@ -95,41 +101,48 @@
...
-end
+
 if basisReuse
     if isfield(LPProblem,'LPBasis')
         basis=LPProblem.LPBasis;
         %use advanced starting information when optimization is initiated.
-        cpxControl.ADVIND=1;
+        if isequal(interface,'tomlab_cplex')
+            cpxControl.advance=1;
+        elseif isequal(interafce,'ILOGcomplex')
```
Comment (laurentheirendt) [inline src/base/solvers/solveCobraLPCPLEX.m]: This typo is one of the reasons why the tests fail. May you fix that? Thanks 😄

**Comment by laurentheirendt** [comment]
@marouenbg, may you have a look at why these tests are failing? Thanks.

**Comment by marouenbg** [comment]
@laurentheirendt all good on my side

**Comment by laurentheirendt** [comment]
Let's restart the job on the ci ...

**Pull request merged**